### PR TITLE
BUMP major upgrade for the static B

### PR DESCRIPTION
**Pull request by benjaminmenetrier**
## Description

Companion PR of https://github.com/JCSDA-internal/saber/pull/99, see this PR for details.

### Issue(s) addressed

None.

## Acceptance Criteria (Definition of Done)

Reasonable modifications of test reference values.

## Dependencies

Waiting on the following PRs:
- [ ] waiting on https://github.com/JCSDA-internal/saber/pull/99

## Impact

None.

## Test Data

None.

### CODE REVIEW

**Comment by danholdaway** [comment]
@travissluka we would like to merge the SABER PR early next week. Can you take a look at this and review?

File: src/soca/Geometry/soca_geom_mod.F90 [hunk 1]
```diff
@@ -180,8 +180,8 @@ subroutine geom_set_atlas_lonlat(self, afieldset)
   ! Create lon/lat field
   afield = atlas_field(name="lonlat", kind=atlas_real(kind_real), shape=(/2,(self%iec-self%isc+1)*(self%jec-self%jsc+1)/))
   call afield%data(real_ptr)
-  real_ptr(1,:) = pack(self%lon(self%isc:self%iec,self%jsc:self%jec),.true.)
-  real_ptr(2,:) = pack(self%lat(self%isc:self%iec,self%jsc:self%jec),.true.)
+  real_ptr(1,:) = reshape(self%lon(self%isc:self%iec,self%jsc:self%jec),(/(self%iec-self%isc+1)*(self%jec-self%jsc+1)/))
+  real_ptr(2,:) = reshape(self%lat(self%isc:self%iec,self%jsc:self%jec),(/(self%iec-self%isc+1)*(self%jec-self%jsc+1)/))
```
Comment (travissluka) [inline src/soca/Geometry/soca_geom_mod.F90:184]: out of curiosity, why were these changed?

**Review by travissluka — APPROVED** [review]

**Comment by benjaminmenetrier** [comment]
This PR can now be merged into `develop`. Thanks!
